styleguide/source/_patterns/02-organisms/05-ec/03-ec-tabs-course-browser/_ec-tabs-course-browser.scss

```diff
@@ -0,0 +1,215 @@
+// Course Browser heading.
+.tabs-course_browser_label {
+  margin: 25px 0;
+  text-align: center;
+}
+
+// Course Browser selector list.
+.tabs-course_browser_list-tabs {
+  border: 1px $gray-8 solid;
+  list-style: none;
+  margin: 0 auto 2em;
+  max-width: 90%;
+  overflow: hidden;
+  padding: 0;
+  position: relative;
+  text-align: center;
+
+  &.open {
+
+    a:active {
+      color: $blue;
+    }
+
+    .icon-select svg {
+      transform: scaleY(-1);
+      filter: FlipV;
+      /* stylelint-disable property-no-vendor-prefix */
+      -ms-filter: "FlipV";
+      /* stylelint-enable */
+    }
+  }
+
+  @include breakpoint($bp-small $bp-med) {
+    max-width: 60%;
+  }
+
+  @include breakpoint($bp-med min-width) {
+    border: 0;
+    height: auto;
+    max-width: 100%;
+    padding: 0;
+
+    &.open {
+      border: 0;
+
+      .icon-select svg {
+        transform: initial;
+        filter: initial;
+        /* stylelint-disable property-no-vendor-prefix */
+        -ms-filter: "initial";
+        /* stylelint-enable */
+      }
+    }
+  }
+}
+
+// Course Browser tab.
+.tabs-course_browser_list_item {
+  border-bottom: 3px transparent solid;
+  color: $black;
+  display: inline-block;
+  line-height: 1.3;
+  padding: 0;
+  margin-bottom: 0;
+  min-width: 190px;
+  text-align: center;
+
+  a {
+    padding: 1em 0.7em;
```

Comment:
(Somehow this didn't get posted with my original comments)

This also needs `display:block` to actually work. I think with that these are have a nice amount of space.
![tabs-padding](https://user-images.githubusercontent.com/238201/27501464-2056d0d0-5833-11e7-8aaf-5e8c8ffda59f.gif)